### PR TITLE
Move all Package related stuff to its own package.

### PR DESCRIPTION
**Pull request by stevana**

### CODE REVIEW

**Comment by stevana** [comment]
Ping?

File: external-resolver/src/main/scala/org/genivi/sota/resolver/packages/PackageDAO.scala [hunk 1]
```diff
@@ -2,17 +2,16 @@
...
-package org.genivi.sota.resolver.db
+package org.genivi.sota.resolver.packages
 
 import org.genivi.sota.refined.SlickRefined._
-import org.genivi.sota.resolver.types.Package
 import slick.driver.MySQLDriver.api._
 
 
-object Packages {
+object PackageDAO {
```
Comment (koshelev) [inline external-resolver/src/main/scala/org/genivi/sota/resolver/packages/PackageDAO.scala]: Let's call it repository. It is a common name for such classes nowadays

File: external-resolver/src/main/scala/org/genivi/sota/resolver/FilterDirectives.scala [hunk 2]
```diff
@@ -61,7 +63,7 @@ class FilterDirectives()(implicit db: Database, mat: ActorMaterializer, ec: Exec
 
   def add(pf: PackageFilter) : Future[PackageFilter] = {
     for {
-      _ <- VehicleFunctions.existsPackage(Package.Id(pf.packageName, pf.packageVersion))
+      _ <- PackageFunctions.exists(Package.Id(pf.packageName, pf.packageVersion))
```
Comment (koshelev) [inline external-resolver/src/main/scala/org/genivi/sota/resolver/FilterDirectives.scala]: Let's keep it simple and call the method `load` :)

File: external-resolver/src/main/scala/org/genivi/sota/resolver/packages/PackageDAO.scala (hunk 1 above)
Comment (stevana) [inline external-resolver/src/main/scala/org/genivi/sota/resolver/packages/PackageDAO.scala]: OK, I'll rename both VehicleDAO and PackageDAO in a separate commit.

**Pull request merged**